README.md

```diff
@@ -54,73 +54,29 @@ Optional fields: `links` and `milestones`
 
 ```json
 {
-  "name": "Eddie Jaoude",
+  "name": "YOUR NAME",
   "type": "personal",
-  "bio": "Founder of EddieHub",
-  "avatar": "https://github.com/eddiejaoude.png",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
   "links": [
     {
       "name": "Follow me on GitHub",
-      "url": "https://github.com/eddiejaoude",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
       "icon": "github"
     },
     {
       "name": "Follow me on Twitter",
-      "url": "https://twitter.com/eddiejaoude",
+      "url": "https://twitter.com/YOUR_TWITTER_USERNAME",
       "icon": "twitter"
-    },
-    {
-      "name": "Learn more about Open Source on my YouTube channel",
-      "url": "https://youtube.com/eddiejaoude",
-      "icon": "youtube"
-    },
-    {
-      "name": "Follow me on LinkedIn",
-      "url": "https://linkedin.com/in/eddiejaoude",
-      "icon": "linkedin"
     }
   ],
   "milestones": [
     {
       "title": "Started Freelancing",
-      "date": "May 2010",
+      "date": "December 2021",
       "icon": "dollar",
       "color": "grey",
-      "description": "Started freelancing again",
-      "url": "https://www.eddiejaoude.io/"
-    },
-    {
-      "title": "Started YouTube",
-      "image": "https://github.com/eddiejaoude.png",
-      "date": "June 2019",
-      "icon": "youtube",
-      "color": "red",
-      "description": "First real video",
-      "url": "https://youtube.com/eddiejaoude"
```

Comment:
Maybe we can add other fields here as an example? People will get an idea of what to add.

Reply:
We could give a link to Eddie's json as a example **if needed**, don't need to add them there 😅

Reply:
> We could give a link to Eddie's json as a example if needed, don't need to add them there 😅

If you give them the link to Eddie's JSON profile, why not add it here then?

Reply:
Because ppl start to use eddie's data if they don't have anything to use, that is the point of this PR 😅

Reply:
> Because ppl start to use eddie's data if they don't have anything to use, that is the point of this PR 😅

Then we can mention that in the readme.